journalpump/senders/websocket.py

```diff
@@ -0,0 +1,286 @@
+from .base import LogSender
+from aiohttp_socks import ProxyConnectionError, ProxyError, ProxyTimeoutError
+from aiohttp_socks.utils import Proxy
+from concurrent.futures import TimeoutError as ConnectionTimeoutError
+from janus import Queue
```

Comment:
This isn't built-in library but there's no dependency to this. Also I would not add this dependency at all. Looking at the implementation it doesn't seem to do anything magical / especially efficient. You can just make `send()` do `asyncio.run_coroutine_threadsafe(self.async_send(messages), self.asyncio_loop).result()` and work with regular queue along with asyncio locks.

Reply:
I ended up removing the, as journalpump does not seem to require one here at this time.